global/config.ts

```diff
@@ -22,52 +22,50 @@
 import getNextConfig from 'next/config';
 
 type AppConfig = {
+  NEXT_PUBLIC_ADMIN_EMAIL: string;
   NEXT_PUBLIC_AUTH_API_ROOT: string;
   NEXT_PUBLIC_AUTH_CLIENT_ID: string;
   NEXT_PUBLIC_AUTH_REDIRECT_URI: string;
   NEXT_PUBLIC_AUTH_SCOPES: string;
   NEXT_PUBLIC_ARRANGER_API_URL: string;
   NEXT_PUBLIC_ARRANGER_GRAPHQL_FIELD: string;
   NEXT_PUBLIC_ARRANGER_INDEX: string;
-  NEXT_PUBLIC_ARRANGER_ADMIN_UI: string;
```

Comment:
removes admin ui... and a bit of code styling/alphabetisation, to avoid duplicates